moveit_core/trajectory_processing/src/ruckig_traj_smoothing.cpp

```diff
@@ -251,10 +251,20 @@ bool RuckigSmoothing::getRobotModelBounds(const double max_velocity_scaling_fact
       ruckig_input.max_acceleration.at(i) = max_acceleration_scaling_factor * DEFAULT_MAX_ACCELERATION;
     }
     // TODO(andyz): Add bounds.jerk_bounded_ for MoveIt1
-    ruckig_input.max_jerk.at(i) = DEFAULT_MAX_JERK;
-    ROS_WARN_STREAM_ONCE_NAMED(LOGNAME, "Joint jerk limits are not defined. Using the default "
-                                            << DEFAULT_MAX_JERK
-                                            << " rad/s^3. You can define jerk limits in joint_limits.yaml.");
+    // For now, we check for a jerk parameter
+    ros::NodeHandle nh;
+    double jerk_limit = DEFAULT_MAX_JERK;
+    if (nh.getParam("ruckig/" + vars.at(i) + "/jerk_limit", jerk_limit))
```

Comment:
where in the moveit config are these being set?

Reply:
I would think it's in a launch file. Wherever the user wants to put it.

Reply:
this clearly needs to be documented. The warning below hints at the `joint_limits.yaml` btw, which doesn't contain this parameter at all. Please either leave a note about fixing this, or do so by providing necessary updates for the resources package, possibly tutorials

Reply:
I'll fix the message below (thanks!) and update the tutorials and (maybe) moveit_resources.

Reply:
here's the documentation PR:  https://github.com/ros-planning/moveit_tutorials/pull/764

I'm going to go ahead and test this one last time then merge it